modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/normalizeDropdownItems.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function normalizeDropdownItems(items) {
+	return items.map(({icon, ...props}) => ({...props, symbolLeft: icon}));
```

Comment:
Can we move the propsTransformer here? 
I think is part of normalization and we don't need a normalized DropdownItems without it.

Something like:

```js
return KBDropdownPropsTransformer(
    items.map(({icon, ...props}) => ({...props, symbolLeft: icon}))
);
```

Maybe isn't a bad idea to add a default value `[]` to items.

Reply:
Yeah makes sense, I had the same feeling but finally left them separated, but it is good to know you think the same. 👍